src/test/prctl/test_prctl.rs

```diff
@@ -0,0 +1,101 @@
+/*
+ * The Shadow Simulator
+ * See LICENSE for licensing information
+ */
+
```

Comment:
I don't think we've generally been including this license header on new rust files, right?

Reply:
Removed.